mirror.php

```diff
@@ -66,14 +66,14 @@
 
 <ul>
  <li>This site is <?php echo is_official_mirror() ? "" : "not"; ?> an official PHP.net mirror site</li>
- <li>The mirror site's address is <?php print_link($MYSITE); ?></li>
+ <li>The mirror site's address is <?php echo make_link($MYSITE); ?></li>
```

Comment:
What about using short echo tags here (and below):
```suggestion
 <li>The mirror site's address is <?=make_link($MYSITE)?></li>
```